src/controllers/customer.js

```diff
@@ -1,53 +1,88 @@
-const { Prisma } = require("@prisma/client")
-const prisma = require('../utils/prisma')
+const { Prisma } = require("@prisma/client");
+const prisma = require("../utils/prisma");
 
 const createCustomer = async (req, res) => {
-    const {
+  console.log("hi tom");
```

Comment:
hi 👋 

Make sure to remove all console logs before submitting a PR in the work environment